example/android/app/src/main/java/com/pspdfkit/flutter/example/FlutterExampleApplication.java

```diff
@@ -1,22 +0,0 @@
-/*
- *   Copyright © 2018-2021 PSPDFKit GmbH. All rights reserved.
- *
- *   THIS SOURCE CODE AND ANY ACCOMPANYING DOCUMENTATION ARE PROTECTED BY INTERNATIONAL COPYRIGHT LAW
- *   AND MAY NOT BE RESOLD OR REDISTRIBUTED. USAGE IS BOUND TO THE PSPDFKIT LICENSE AGREEMENT.
- *   UNAUTHORIZED REPRODUCTION OR DISTRIBUTION IS SUBJECT TO CIVIL AND CRIMINAL PENALTIES.
- *   This notice may not be removed from this file.
- */
-package com.pspdfkit.flutter.example;
-
-import android.content.Context;
-
-import androidx.multidex.MultiDex;
-import io.flutter.app.FlutterApplication;
-
-public class FlutterExampleApplication extends FlutterApplication {
-    @Override
-    protected void attachBaseContext(Context base) {
-        super.attachBaseContext(base);
-        MultiDex.install(this);
-    }
-}
```

Comment:
I'm uncertain about this. Have you tested the PR with an API 19 emulator as well? By removing `MultiDex.install(this)` from our application class, we don't support multidex on devices that don't have system support for multiple dex files. I expect that this means, our example won't run on API 19 anymore (API 21 is the first version that doesn't require the multidex support library anymore). Since we specify the `androidMinSdkVersion` as 19 in our `config.gradle`, this seems problematic.

The simplest solution would be to state inside the app's manifest to use the `MultiDexApplication` class, as described here: https://developer.android.com/studio/build/multidex

```xml
<?xml version="1.0" encoding="utf-8"?>
<manifest xmlns:android="http://schemas.android.com/apk/res/android"
    package="com.example.myapp">
    <application
            android:name="androidx.multidex.MultiDexApplication" >
        ...
    </application>
</manifest>
```

Reply:
Ah great thanks for the info. I wasn't sure of the implications of removing this, but it seemed to be necessary to not use `io.flutter.app.FlutterApplication` since the 1.12 update. Although I have since checked this and it's not a problem. I'll add the file back in as it is probably clearer to do it this way and more in-keeping with how android applications are created.